<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>11</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mainpage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="TitleLabel" name="TitleLabel">
         <property name="text">
          <string>LOGIN</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="ImageLabel" name="logo">
            <property name="text">
             <string>dsa</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="BodyLabel" name="BodyLabel">
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="LineEdit" name="ledit_username"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="BodyLabel" name="BodyLabel_2">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PasswordLineEdit" name="ledit_pw"/>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="PushButton" name="btn_guest">
         <property name="text">
          <string>Login as guest</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PrimaryPushButton" name="btn_login">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="guestpage">
      <widget class="TitleLabel" name="TitleLabel_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>GUEST</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Ledit_NIM_g">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>90</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="PushButton" name="PushButton_2">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>120</y>
         <width>102</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>search</string>
       </property>
      </widget>
      <widget class="SubtitleLabel" name="SubtitleLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>119</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>data</string>
       </property>
      </widget>
      <widget class="PushButton" name="btn_logout2">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>20</y>
         <width>102</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Logout</string>
       </property>
       <property name="iconSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </widget>
      <widget class="BodyLabel" name="BodyLabel_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>170</y>
         <width>63</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>nama</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="BodyLabel_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>200</y>
         <width>63</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>prodi</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="BodyLabel_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>230</y>
         <width>63</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>NIM</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="BodyLabel_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>260</y>
         <width>63</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>skor</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="nama_g">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>471</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Body label</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="prodi_g">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>200</y>
         <width>471</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Body label</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="nim_g">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>230</y>
         <width>471</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Body label</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="skor_g">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>260</y>
         <width>471</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Body label</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="userpage">
      <widget class="PushButton" name="PushButton_12">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>120</y>
         <width>102</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="PushButton" name="btn_logout">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>20</y>
         <width>102</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Logout</string>
       </property>
       <property name="iconSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="leditSearchNIM_6">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>90</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Cari NIM</string>
       </property>
      </widget>
      <widget class="TitleLabel" name="TitleLabel_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nama</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="BodyLabel_46">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>63</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>prodi</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="BodyLabel_47">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>63</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>skor</string>
       </property>
      </widget>
      <widget class="TitleLabel" name="TitleLabel_11">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>0</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>(#peringkat)</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="BodyLabel_48">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>63</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>nim</string>
       </property>
      </widget>
      <widget class="TableWidget" name="TableWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>551</width>
         <height>231</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::DoubleClicked</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>10</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>167</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Nama</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Prodi</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Skor</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="adminpage_3">
      <widget class="PushButton" name="PushButton_4">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>60</y>
         <width>102</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>search</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="prodi_a">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>190</y>
         <width>451</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>tesdata</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="nama_a">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>451</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>tesdata</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="BodyLabel_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>34</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>skor</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="BodyLabel_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>34</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>prodi</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="BodyLabel_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>34</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>nim</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="skor_a">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>220</y>
         <width>451</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>tesdata</string>
       </property>
      </widget>
      <widget class="SubtitleLabel" name="SubtitleLabel_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>119</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>data</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="nim_a">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>160</y>
         <width>451</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>tesdata</string>
       </property>
      </widget>
      <widget class="BodyLabel" name="BodyLabel_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>34</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>nama</string>
       </property>
      </widget>
      <widget class="SubtitleLabel" name="SubtitleLabel_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>119</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>ADMIN</string>
       </property>
      </widget>
      <widget class="ComboBox" name="ComboBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>511</width>
         <height>32</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>511</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">ComboBox {
    border: 1px solid rgba(0, 0, 0, 0.073);
    border-radius: 5px;
    border-bottom: 1px solid rgba(0, 0, 0, 0.183);
    padding: 5px 31px 6px 11px;
    /* font: 14px 'Segoe UI', 'Microsoft YaHei', 'PingFang SC'; */
    color: black;
    background-color: rgba(255, 255, 255, 0.7);
    text-align: left;
}

ComboBox:hover {
    background-color: rgba(249, 249, 249, 0.5);
	adjustSize()
}

ComboBox:pressed {
    background-color: rgba(249, 249, 249, 0.3);
    border-bottom: 1px solid rgba(0, 0, 0, 0.073);
    color: rgba(0, 0, 0, 0.63);
	adjustSize()
}

ComboBox:disabled {
    color: rgba(0, 0, 0, 0.36);
    background: rgba(249, 249, 249, 0.3);
    border: 1px solid rgba(0, 0, 0, 0.06);
    border-bottom: 1px solid rgba(0, 0, 0, 0.06);
}
</string>
       </property>
      </widget>
      <widget class="PushButton" name="PushButton">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>330</y>
         <width>102</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Update Skor</string>
       </property>
      </widget>
      <widget class="PushButton" name="btn_logout3">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>20</y>
         <width>102</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Logout</string>
       </property>
       <property name="iconSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </widget>
      <widget class="LineEdit" name="Ledit_NIM">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>211</width>
         <height>33</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PasswordLineEdit</class>
   <extends>LineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
